server/src/definitions/YmlDefinitionProvider.ts

```diff
@@ -2,12 +2,14 @@ import { Definition, Location } from 'vscode-languageserver';
 
 import { AbstractYmlObject } from '../yml-objects';
 
+export const LOCATION_FILTER = (uri: string) => (defLoc: AbstractYmlObject) => defLoc.definitionLocation.uri !== uri;
```

Comment:
Not trivial to guess that this _excludes_ the items that bear the given URI. Docs? Maybe also longer name but not sure about this.